cluster/engine.go

```diff
@@ -1520,9 +1520,9 @@ func IsConnectionError(err error) bool {
 	// docker/api returns ErrConnectionFailed error, so we check for that as long as dockerclient exists
 	return err == dockerclient.ErrConnectionRefused ||
 		engineapi.IsErrConnectionFailed(err) ||
-		strings.Contains(err.Error(), "onnection refused") ||
-		strings.Contains(err.Error(), "annot connect to the docker engine endpoint") ||
-		strings.Contains(err.Error(), "annot connect to the Docker daemon")
+		strings.Contains(err.Error(), "connection refused") ||
+		strings.Contains(err.Error(), "cannot connect to the docker engine endpoint") ||
+		strings.Contains(err.Error(), "cannot connect to the Docker daemon")
```

Comment:
@YuPengZTE I think it was done like this because there were client code changed between upper/lower cases. To be compatible I removed the first letter. Just adding back the first letter may break older versions. I'd suggest not making this change, or adding both upper/lower cases.